config/config.exs

```diff
@@ -73,10 +73,9 @@ config :fz_http,
   openid_connect_providers: "{}",
   saml_identity_providers: %{},
   saml_entity_id: "urn:firezone.dev:firezone-app",
-  saml_certfile_path: "apps/fz_http/priv/cert/saml_selfsigned.pem",
-  saml_keyfile_path: "apps/fz_http/priv/cert/saml_selfsigned_key.pem",
+  saml_certfile_path: "#{git_root}/apps/fz_http/priv/cert/saml_selfsigned.pem",
```

Comment:
I think this might break the Docker build because we [move the app bundle]() after `mix release`. Could this be overridden perhaps in `config/runtime.exs` with something like

```suggestion
  saml_certfile_path: Path.expand("../apps/fz_http/priv/cert/saml_selfsigned.pem",__DIR__),
```

Reply:
e.g.

```
firezone-firezone-1  | [error] [Samly] Failed load SP certfile ["/apps/fz_http/priv/cert/saml_selfsigned.pem"]: %{certfile: "/apps/fz_http/priv/cert/saml_selfsigned.pem", entity_id: "urn:firezone.dev:firezone-app", id: "firezone", keyfile: "/apps/fz_http/priv/cert/saml_selfsigned_key.pem"}
firezone-firezone-1  | [error] [Samly] Failed load SP keyfile ["/apps/fz_http/priv/cert/saml_selfsigned_key.pem"]: %{certfile: "/apps/fz_http/priv/cert/saml_selfsigned.pem", entity_id: "urn:firezone.dev:firezone-app", id: "firezone", keyfile: "/apps/fz_http/priv/cert/saml_selfsigned_key.pem"}
```

```
firezone-firezone-1  | fatal: not a git repository (or any of the parent directories): .git
```

Got a fix coming in another PR

Reply:
Good catch, I forgot that path expansion would work relatively to the umbrella root!

Reply:
Added an issue to fix it long term https://github.com/firezone/firezone/issues/1229